lib/tasks/script.rake

```diff
@@ -0,0 +1,7 @@
+namespace :script do
+  desc 'Run all pending onetime scripts'
+  task run_pending_onetime: :environment  do
+    runner = Grnds::Onetime::Runner.new
+    runner.run_all
+  end
```

Comment:
There should be another task here like check_table_exists and then this is `task run_pending_onetime: :check_table_exists` and then `task check_table_exists: :environment`